cg/models/demultiplex/valid_indexes.py

```diff
@@ -0,0 +1,10 @@
+import logging
```

Comment:
This file is misleading. Found in valid_indexes but contains no indices, but instead only one model!

Reply:
What's the suggestion?

Reply:
Move model to live among its kind, delete file

Reply:
Sure I'll do that! I'm a bit confused since I have been asked in other reviews to move models from where it is used to the `models` module.